packages/controls/src/controls/shape/v2/shape-v2.test.ts

```diff
@@ -34,388 +30,152 @@ function deserializer(record: DeserializedRecord): ControlDefinition {
 }
 
 describe('ShapeV2', () => {
-  describe('constructor', () => {
-    test('heterogenous controls', () => {
-      const shape = ShapeV2({
+  test('deserializes to `Group` with `shape-v2::v1` data type', () => {
```

Comment:
A new set of "ShapeV2" tests here that is complimentary to the `Group` tests. 

I put "ShapeV2" in quotes because there is no separate "ShapeV2" control definition, it's just a `Group` with a 'ShapeV2' data tag.